forms-flow-data-layer/src/config/envs.py

```diff
@@ -66,64 +66,77 @@ def construct_postgres_uri(
 
 class ENVS:
     # FormIO MongoDB Configuration
-    FORMIO_MONGO_HOST: str = os.getenv("FORMIO_MONGO_HOST", "localhost")
-    FORMIO_MONGO_PORT: str = os.getenv("FORMIO_MONGO_PORT", "27017")
-    FORMIO_MONGO_USERNAME: str = os.getenv("FORMIO_MONGO_USERNAME", "")
-    FORMIO_MONGO_PASSWORD: str = os.getenv("FORMIO_MONGO_PASSWORD", "")
-    FORMIO_MONGO_DATABASE: str = os.getenv("FORMIO_MONGO_DATABASE", "formio")
-    FORMIO_MONGO_OPTIONS: str = os.getenv("FORMIO_MONGO_OPTIONS", "")
+    FORMIO_DB_HOST: str = os.getenv("FORMIO_MONGO_HOST", "localhost")
```

Comment:
Please make sure the env vars match with what's provided in corresponding services

Reply:
resolved